pkg/apis/submariner.io/v1/types.go

```diff
@@ -70,15 +70,26 @@ type EndpointSpec struct {
 	ClusterID string `json:"cluster_id"`
 	CableName string `json:"cable_name"`
 	// +optional
-	HealthCheckIP    string            `json:"healthCheckIP,omitempty"`
-	Hostname         string            `json:"hostname"`
-	Subnets          []string          `json:"subnets"`
-	PrivateIP        string            `json:"private_ip"`
-	PublicIP         string            `json:"public_ip"`
-	NATEnabled       bool              `json:"nat_enabled"`
-	Backend          string            `json:"backend"`
-	BackendConfig    map[string]string `json:"backend_config,omitempty"`
-	NATDiscoveryPort *int32            `json:"natDiscoveryPort,omitempty"`
+	HealthCheckIP string            `json:"healthCheckIP,omitempty"`
+	Hostname      string            `json:"hostname"`
+	Subnets       []string          `json:"subnets"`
+	PrivateIP     string            `json:"private_ip"`
+	PublicIP      string            `json:"public_ip"`
+	NATEnabled    bool              `json:"nat_enabled"`
+	Backend       string            `json:"backend"`
+	BackendConfig map[string]string `json:"backend_config,omitempty"`
+}
+
+const (
+	BackendConfigPrefix            = "gateway.submariner.io/"
+	UDPPortBackendConfig           = "udp-port"
+	NATTDiscoveryPortBackendConfig = "natt-discovery-port"
```

Comment:
I think we can omit `Backend` from the names. Frankly I would've preferred `Backend` -> `CableDriver` and `BackendConfig` -> `CableDriverConfig` 😄 

```suggestion
	GatewayConfigLabelPrefix            = "gateway.submariner.io/"
	UDPPortConfig           = "udp-port"
	NATTDiscoveryPortConfig = "natt-discovery-port"
```

Reply:
> I think we can omit `Backend` from the names. Frankly I would've preferred `Backend` -> `CableDriver` and `BackendConfig` -> `CableDriverConfig`

yup, agreed.

Reply:
Done